graph/tags/tags.go

```diff
@@ -6,8 +6,10 @@ import (
 	"github.com/docker/distribution/registry/api/v2"
 )
 
+// DEFAULTTAG (latest) is used when tag is not specified.
 const DEFAULTTAG = "latest"
```

Comment:
this should be `DefaultTag`

`DefaultTag defines the default tag used when performing images related actions and no tag strings is specified`
maybe @stevvooe has a better comment for this tho